src/components/Header/index.tsx

```diff
@@ -46,7 +46,7 @@ const Header: React.FC = () => {
       <strong>Rocket Redis</strong>
 
       {shouldUseMacOSWindowActions ? (
-        <WindowActions position="left">
+        <WindowActions position="left" isMac>
```

Comment:
I think this one should be `showIconsOnHover` instead of `isMac`, so the CSS does not depends on external interfaces like the OS and we stay more declarative.